diff --git "a/C:\\Users\\cedric\\AppData\\Local\\Temp\\sid1303.tmp\\side-post-462b468-left.php" "b/C:\\wamp\\vhosts\\wpapp\\wp-content\\plugins\\plugin\\posts-to-posts\\core\\side-post.php"
index c84a3f9..a504f8d 100644
--- "a/C:\\Users\\cedric\\AppData\\Local\\Temp\\sid1303.tmp\\side-post-462b468-left.php"
+++ "b/C:\\wamp\\vhosts\\wpapp\\wp-content\\plugins\\plugin\\posts-to-posts\\core\\side-post.php"
@@ -68,7 +68,7 @@ class P2P_Side_Post extends P2P_Side {
 
 	function can_edit_connections() {
 		try {
-			return current_user_can( $this->get_ptype()->cap->edit_posts );
+			return current_user_can( $this->get_ptype()->cap->read );
 		} catch ( P2P_Exception $e ) {
 			trigger_error( $e->getMessage(), E_USER_WARNING );
 			return false;
